Dockerfile

```diff
@@ -9,6 +9,10 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && apt-get -y upgrade
 RUN apt-get install -y software-properties-common
 
+# PostgreSQL apt server for newer PostgreSQL and PostGIS versions
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8
+RUN add-apt-repository "deb http://apt.postgresql.org/pub/repos/apt/ bionic-pgdg main"
```

Comment:
Ik krijg hier warnings:

```
Step 7/21 : RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8
 ---> Running in a65e4308526e
Warning: apt-key output should not be parsed (stdout is not a terminal)
Executing: /tmp/apt-key-gpghome.W5W5zSFQ79/gpg.1.sh --keyserver keyserver.ubuntu.com --recv-keys B97B0AFCAA1A47F044F244A07FCC7D46ACCC4CF8
gpg: key 7FCC7D46ACCC4CF8: 6 signatures not checked due to missing keys
gpg: key 7FCC7D46ACCC4CF8: public key "PostgreSQL Debian Repository" imported
gpg: Total number processed: 1
gpg:               imported: 1
```

Is dat nog op te lossen?

Reply:
Zou zo geen warning moeten geven.